drivers/axi_core/jesd204/axi_jesd204_rx.c

```diff
@@ -58,6 +58,10 @@
 #define JESD204_RX_REG_SYNTH_NUM_LANES			0x10
 #define JESD204_RX_REG_SYNTH_DATA_PATH_WIDTH	0x14
 
+#define JESD204_RX_REG_SYNTH_REG_1		0x18
+#define JESD204_RX_ENCODER_MASK			GENMASK(9, 8)
+#define JESD204_RX_ENCODER_GET(x)	field_get(JESD204_RX_ENCODER_MASK, x)
```

Comment:
if you have this utilities as functions should we just use them as they are ? I don't particularly like masking functions with macros.

Reply:
This seems to be a common approach in Linux kernel - however, it's true that most of those functions are GCC built-in ones.
In this particular case, I just want to use JESD204_RX_ENCODER_GET() the same way as it is used in the Linux kernel.